src/main/java/com/fmgs/conferenceapp/controller/PaperController.java

```diff
@@ -0,0 +1,59 @@
+package com.fmgs.conferenceapp.controller;
+
+import com.fmgs.conferenceapp.model.Paper;
+import com.fmgs.conferenceapp.model.Qualifiers;
+import com.fmgs.conferenceapp.model.Review;
+import com.fmgs.conferenceapp.repository.PaperRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Map;
+
+@RestController
+public class PaperController {
+
+    private PaperRepository repository;
+    private Review reviewCtrl;
+
+    @Autowired
+    public PaperController(PaperRepository repository) {
+        this.repository = repository;
+    }
+
+    @GetMapping("/papers")
+    @CrossOrigin(origins = "*")
+    public List<Paper> getAllPapers() {
+        return repository.findAll();
+    }
+
+    @PostMapping("papers/post")
+    @CrossOrigin(origins = "*")
+    public void postPaper(@RequestBody Paper paper) {
+        repository.save(paper);
+    }
+
+    @PostMapping("papers/review")
+    @CrossOrigin(origins = "*")
+    public Paper reviewPaper(@RequestBody Map<String, String> params, String reviewerID, int reviewInt){
+        Paper toBeReviewed = repository.getPaperById(params.get("id"));
+        Review rev = new Review();
+        rev.addReview(reviewerID, reviewInt);
```

Comment:
Nu ai nevoie sa creezi un obiect de tip review de fiecare data.. El se creeaza in constructor.. Daca faci asta, la fiecare review pe care îl lași, reînnoiești lista, deci rămâi cu un singur review.. Si noi vrem sa avem 4. Aici trebuia sa folosești getterul si setterul